test/test_genome.py

```diff
@@ -12,7 +12,7 @@
 import numpy as np
 import pandas as pd
 
-from cnvlib import read
+from cnvlib import read_ga as read
```

Comment:
Make sure to actually test `GenomicArray` not `CopyNumArray`

Reply:
How about importing `read_ga` with is original name and updating the function calls within the unit tests to `read_ga`?